pkg/model/labeler.go

```diff
@@ -277,7 +277,13 @@ func (l *Labeler) GetSelectorHostScope(host *chi.ChiHost) map[string]string {
 
 // appendCHILabels appends CHI-provided labels to labels set
 func (l *Labeler) appendCHILabels(dst map[string]string) map[string]string {
-	return util.MergeStringMapsOverwrite(dst, l.chi.Labels)
+	sourceLabels := l.chi.Labels
+	if len(l.chop.Config().LabelsIgnore) > 0 {
+		sourceLabels = util.MergeStringMapsOverwrite(map[string]string{}, sourceLabels)
```

Comment:
Probably, it would be clearer to introduce new util function to make a Copy of a map.
Merging over an empty map is not that obvious

Reply:
Good point, that makes for cleaner code. I added two helper functions. (`CopyMap` and `CopyMapExcept`)